README.md

```diff
@@ -26,4 +26,16 @@ This command generates static content into the `build` directory and can be serv
 
 ## Deployment
 
-Deployment happens in GitHub Actions to GitHub Pages
\ No newline at end of file
+Deployment happens in GitHub Actions to GitHub Pages
+
+## Versioning
+
+To add a new version, edit the files in `docs` folder with the new version content. Once the new version content is finalized, run:
+
+```console
+yarn run docusaurus docs:version <version number>
+```
+
+This will copy the files in the `docs` folder to `versioned_docs/version-<version number>/`, create a `version-<version number>-sidebar.json` file under `version_sidebars` and add the version number to the `version.json` file. 
+
+To edit the files in a previous version, edit the files from `versioned_docs` directly. To edit files in the latest version, both the files under `versioned_docs` and `docs` need to be updated. Thus, it's good practice to create the version only when the content in the `docs` folder is finalized.
```

Comment:
is this because only versioned docs are shown on the deployed docs? or would there ever be a case where it would make sense to update the latest docs only (to reflect developed, but not yet released features in honeycomb)?

Reply:
Yes that's the case. Once the versions are set, only the files in the version_docs will be displayed. Docs will just be there to create a version from, so it would be used just like in the case you mentioned.